content/zh-cn/examples/application/mysql/mysql-configmap.yaml

```diff
@@ -4,15 +4,14 @@ metadata:
   name: mysql
   labels:
     app: mysql
+    app.kubernetes.io/name: mysql
 data:
   primary.cnf: |
-    # Apply this config only on the primary.
+    # 仅在主服务器上应用此配置
```

Comment:
Could I ask why the period `。` is removed from Chinese? Thanks.

Reply:
It doesn't have to be removed. Keep it there if you wish.
My feeling is that a period in this situation looks weird.